addons/website_quote/controllers/main.py

```diff
@@ -95,14 +95,6 @@ def view(self, order_id, pdf=None, token=None, message=False, **post):
                 [('partner_id', '=', order_sudo.partner_id.id),
                 ('acquirer_id', 'in', [acq.id for acq in values['s2s_acquirers']])])
 
-            for acq in values['form_acquirers']:
-                acq.form = acq.render('/', order_sudo.amount_total, order_sudo.pricelist_id.currency_id.id,
```

Comment:
It is unused since this commit (commits 06fe263a150f42ec36c91e6219fdc8f19fd50798).